### ISSUE TITLE
Incompatibility with Crafting Tweaks

### ISSUE THREAD

**Issue by Crystallyne** [SlimeKnights/TinkersConstruct#5004]
### Minecraft Version

1.18.2

### Forge Version

40.1.68

### Mantle Version

Unknown

### Tinkers' Construct Version

3.5.2.40

### Describe your issue

[Crafting Tweaks](https://www.curseforge.com/minecraft/mc-mods/crafting-tweaks) adds buttons to the crafting window for common tasks. The Crafting Station for this version of TC does not show those buttons. They show up fine on standard crafting tables. I'm not sure which mod needs to be updated, so I'm reporting the issue on both mods.

### Other mods

[Crafting Tweaks](https://www.curseforge.com/minecraft/mc-mods/crafting-tweaks)

### Tried reproducing with just Tinkers?

No

### Performance Enchancers

None of the above

### Searched for known issues?

Searched open issues

**Comment by KnightMiner** [comment]
Have you reported this to crafting tweaks? If so, link the issue here.

**Comment by Crystallyne** [comment]
Yes, I posted a comment on the Crafting Tweaks page. You can find my post here: https://www.curseforge.com/minecraft/mc-mods/crafting-tweaks?comment=137

Edit: Didn't mean to mark this as closed.

**Comment by KnightMiner** [comment]
A comment will get lost. Use the issues link for crafting tweaks to report a proper bug report on their github: https://github.com/ModdingForBlockheads/CraftingTweaks/issues

Looks like there is one, ModdingForBlockheads/CraftingTweaks#151

**Comment by Crystallyne** [comment]
Oh perfect, thank you. Sorry, I didn't think to click the Issues button on the Crafting Tweaks page on Curse. Do you still need me to create a new post to report the issue? I would assume that one would suffice?

**Comment by KnightMiner** [comment]
no reason to make duplicates

**Comment by Crystallyne** [comment]
That's what I thought. Thank you for the help.

**Comment by KnightMiner** [comment]
Crafting Tweaks support was added in #5277, which is in the latest 1.19.2 and 1.20.1 builds.
